modules/standard/CommDiagnostics.chpl

```diff
@@ -284,6 +284,21 @@ module CommDiagnostics
      */
     var cache_put_misses: uint(64);
 
+    // NOTE: these counters are attached to the lifetime of the data/network operations,
+    // so nothing is remembered with Aout.
```

Comment:
This comment was useful to us during development but I think you should remove it from this PR.